src/main/java/io/cdap/plugin/WindowAggregationConfig.java

```diff
@@ -18,13 +18,13 @@
 
 import com.google.common.base.Joiner;
 import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
```

Comment:
why did we add the guava dependency? was there some issue with org.apache.parquet.Strings?

Reply:
@rmstar because we changed the scope of the dependency ``spark-repl_2.11`` to provided as required, we encounter issue in validation action, 

``CDAP 6.3.0 : Missing class com.google.common.base.Splitter. It may be caused by missing dependency jar(s) in the artifact jar.`` 

so we added guava dependency, and after that we got : 

``Missing class org.apache.parquet.Strings. It may be caused by missing dependency jar(s) in the artifact jar.`` so we had to change to  ``com.google.common.base.Strings``